_studio/shared/umc/codec/h265_dec/src/umc_h265_bitstream_headers.cpp

```diff
@@ -981,7 +981,9 @@ void H265HeadersBitstream::parseVUI(H265SeqParamSet *pSPS)
     pSPS->frame_field_info_present_flag = Get1Bit();
 
     pSPS->default_display_window_flag = Get1Bit();
-    if (pSPS->default_display_window_flag)
+    //Hack to ignore the video cropping here.
+    //if (pSPS->default_display_window_flag)
+    if (0)
```

Comment:
if (0) is not good coding. And could you please explain the reason?

Reply:
Just for a manual build, will not check in.

Reply:
Please add [DO NOT MERGE] at the title of PR and commit for avoiding misunderstanding. Thanks.